### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by Snakeybus** [amiinoo/AGB-buttontest#1]
my EZ FLASH cant read it and its a rom player

**Comment by heroldev** [comment]
make sure you put the *.gba file in the release on your EZ Flash

also is this on an ez-flash 3in1, ez-flash omega, or ez-flash IV/Reform?

**Comment by SituatedAction** [comment]
I've been using this ROM a _long_ time and a huge proponent of what you've made. I've been running it on an actual cart I flashed it to without any issues... made that cart in 2019. I also have it loaded onto a Krikzz Everdrive GBA X5 without any issue and also an EZ-Flash Omega without any issues either. A friend of mine just made an actual cart with the new version, and it's also working fine.

In short I am not finding anything special needed for this ROM; just load it up and go.

**Comment by remy** [comment]
I've seen something similar with a my Everdrive X5. Took me a while to work it out, but if I disabled the Everdrive's "Quick Boot" (so the initial GBA boot logo is shown), the ROM loaded correctly.

Without this, the ROM would boot, but I'd see a black screen whilst only seeing the I-bar sliding down and up, then the screen goes black.

**Comment by alexfornuto** [comment]
Similar to:

> Everdrive's "Quick Boot"

I found that with an EZ Flash Definitive Edition it works with a "clean boot", just not the "boot with addon" option. Just sharing for the next Googler.

**Comment by heroldev** [comment]
Odds are the flashcart probably makes a preliminary ROM patch that causes the program to break. I don't have an Omega, Definitive Edition, or Everdrive to test with, only an EZFlash IV and 3-in-1, so I'm going to mark this closed with the fix referenced above. 

Make sure to disable automatic ROM patching when trying to boot this, as referenced in https://github.com/heroldev/AGB-buttontest/issues/1#issuecomment-1741876561 and https://github.com/heroldev/AGB-buttontest/issues/1#issuecomment-1195249645
